src/vue/forms/WithdrawalForm.vue

```diff
@@ -0,0 +1,356 @@
+                  'withdrawal-form.balance' | globalize({
+                    amount: balanceInfo.balance,
+                    asset: form.assetCode
+                  })
+                }}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="app__form-row withdrawal__form-row">
+          <input-field
+            white-autofill
+            class="app__form-field"
+            v-model.trim="form.amount"
+            type="number"
+            @blur="touchField('form.amount')"
+            :label="'withdrawal-form.amount' | globalize({
+              asset: form.assetCode
+            })"
+            :disabled="formMixin.isDisabled"
+            :error-message="getFieldErrorMessage('form.amount', {
+              from:MIN_AMOUNT,
+              to:balanceInfo.balance
+            })"
+          />
+        </div>
+
+        <div class="app__form-row withdrawal__form-row">
+          <input-field
+            white-autofill
+            class="app__form-field"
+            v-model.trim="form.address"
+            @blur="touchField('form.address')"
+            :error-message="getFieldErrorMessage('form.address')"
+            :label="'withdrawal-form.destination-address' | globalize({
+              asset: form.assetCode
+            })"
+            :monospaced="true"
+            :disabled="formMixin.isDisabled"
+          />
+        </div>
+
+        <div class="app__form-row withdrawal__form-row">
+          <table class="withdrawal__fee-table">
+            <tbody
+              class="withdrawal__fee-tbody"
+              :class="{ 'withdrawal__data_loading': isFeesLoadPending }">
+              <tr>
+                <td>{{ 'withdrawal-form.network-fee-hint' | globalize }}</td>
+                <td>-</td>
+              </tr>
+              <tr>
+                <td>{{ 'withdrawal-form.fixed-fee' | globalize }}</td>
+                <td>
+                  <!-- eslint-disable-next-line max-len -->
+                  {{ { value: fixedFee, currency: form.assetCode } | formatMoney }}
+                </td>
+              </tr>
+              <tr>
+                <td>{{ 'withdrawal-form.percent-fee' | globalize }}</td>
+                <td>
+                  <!-- eslint-disable-next-line max-len -->
+                  {{ { value: percentFee, currency: form.assetCode } | formatMoney }}
+                </td>
+              </tr>
+            </tbody>
+            <tbody class="withdrawal__total-fee-tbody">
+              <tr>
+                <td>
+                  {{ 'withdrawal-form.total-amount-account' | globalize }}
+                </td>
+                <td>
+                  <!-- eslint-disable-next-line max-len -->
+                  {{ { value: (+percentFee + +fixedFee), currency: form.assetCode } | formatMoney }}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
+        <div class="app__form-actions withdrawal__action-margin">
```

Comment:
the same about confusing names. it seems I've missed it when was reviewing withdrawal

Reply:
ok